src/alarms/notifications/App.js

```diff
@@ -58,21 +59,122 @@ class App extends Component {
       .then(response => response.json())
       .then(data => {
         this.setState({
-          isFetching: false,
-          total: data.count,
-          alarms: data.results,
+          isFetchingRasterAlarms: false,
+          total: this.state.total + data.count,
+          alarms: this.state.alarms.concat(
+            data.results.map(e => {
+              e.type = "RasterAlarm";
+              return e;
+            })
+          ),
+          page: page
+        });
+      });
+  }
+
+  loadTimeseriesAlarms(page, newProps) {
+    const { selectedOrganisation } = newProps || this.props;
+    const organisationId = selectedOrganisation.uuid;
+
+    fetch(
+      `/api/v3/timeseriesalarms/?page=${page}&organisation__unique_id=${organisationId}`,
+      {
+        credentials: "same-origin"
+      }
+    )
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          isFetchingTimeseriesAlarms: false,
+          total: this.state.total + data.count,
+          alarms: this.state.alarms.concat(
+            data.results.map(e => {
+              e.type = "TimeseriesAlarm";
+              return e;
+            })
+          ),
           page: page
```

Comment:
De timeseries en raster alarmen worden nu beide gefetched.
Het is niet deterministisch welke als eerste terugkomt. Daarom mag nooit het hele total en alrmen veld overschreven worden, maar moet het bij de bestaande waarde worden opgeteld.
Dit comment geld ook voor volgend blok